roomee-api/roomee-api/Controllers/EventController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Data.SqlClient;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Threading;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using Newtonsoft.Json;
+using Microsoft.AspNetCore.JsonPatch.Adapters;
+using roomee_api.Models;
+using roomee_api.Utilities;
+
+namespace roomee_api.Controllers{
+    [Route("v1/event")]
+    [ApiController]
+    public class EventController : ControllerBase{
+        [HttpGet]
+        public IActionResult GetEvent([FromQuery][Required] string type, [FromQuery][Required] string identifier){
```

Comment:
This function takes in a type and an identifier. As far as I can see, the Event is only looked up by id and nothing else. In this case, you would only need to take in a single integer for the id as it is the only method for getting a Room. Then, the [HttpGet] would become [Http("{id}")].